sw/device/lib/log.h

```diff
@@ -141,7 +141,7 @@
  */
 #ifndef PRINT_LOG
 #warning PRINT_LOG undefined: it will result in vacuous invocation
-#define PRINT_LOG(log_header, fmt, ...)
+#define PRINT_LOG(log_header, ...)
```

Comment:
does removing fmt here change the behavior of the logging function..?

Reply:
It was intentional to ensure that zero-length varargs don't get built.  The varargs now have the format string in them and this gets used as the first argument.

Reply:
@tjaychen do you have your own implementations of `PRINT_LOG` or the other similar macros in this file? If so, their signature may need to be updated, but the functionality should not be different.